lib/mix/test/mix/tasks/compile.yecc_test.exs

```diff
@@ -1,30 +1,36 @@
-Code.require_file "../../test_helper.exs", __DIR__
+Code.require_file("../../test_helper.exs", __DIR__)
 
 defmodule Mix.Tasks.Compile.YeccTest do
   use MixTest.Case
   import ExUnit.CaptureIO
 
   setup do
-    Mix.Project.push MixTest.Case.Sample
+    Mix.Project.push(MixTest.Case.Sample)
     :ok
   end
 
   test "compilation continues if one file fails to compile" do
     in_fixture "compile_yecc", fn ->
       file = Path.absname("src/zzz.yrl")
-      File.write! file, """
+
+      File.write!(file, """
       oops.
-      """
-
-      capture_io fn ->
-        assert {:error, [%Mix.Task.Compiler.Diagnostic{
-          compiler_name: "yecc",
-          file: ^file,
-          message: "syntax error before: '.'",
-          position: 1,
-          severity: :error
-        }]} = Mix.Tasks.Compile.Yecc.run(["--force"])
-      end
+      """)
+
+      capture_io(fn ->
+        assert {
+                 :error,
+                 [
+                   %Mix.Task.Compiler.Diagnostic{
+                     compiler_name: "yecc",
+                     file: ^file,
+                     message: "syntax error before: '.'",
+                     position: 1,
+                     severity: :error
+                   }
+                 ]
+               } = Mix.Tasks.Compile.Yecc.run(["--force"])
```

Comment:
This is exactly the stuff that needs to be refactored in order to look good. Write it like this:

```elixir
assert {:error, [diagnostic]} = Mix.Tasks.Compile.Yecc.run(["--force"])
assert %Mix.Task.Compiler.Diagnostic{
  ...
} = diagnostic
```

and see what the formatter spits out :)